src/components/Modal/Modal.tsx

```diff
@@ -89,6 +87,10 @@ export function Modal({
         onOutsideClick,
         onClose,
         contentRefs: [contentRef],
+        enableFocusTrap,
+        focusTrapContainersRefs: [contentRef],
+        focusTrapOptions,
```

Comment:
I guess, these options should depend on `disableEscapeKeyDown`, `disableOutsideClick` props